virttest/qemu_devices/qcontainer.py

```diff
@@ -2641,10 +2641,9 @@ def memory_object_define_by_params(self, params, name):
         'memory-backend-ram'.
         """
         params = params.object_params("mem")
-        backend = params.get("backend_mem", "memory-backend-ram")
-        attrs = qdevices.Memory.__attributes__[backend][:]
-        params = params.copy_from_keys(attrs)
         params.setdefault("backend", "memory-backend-ram")
+        attrs = qdevices.Memory.__attributes__[params["backend"]][:]
```

Comment:
It is after `params = params.object_params("mem")`, good ~

Reply:
@huangyum Would you split it into a separated commit if possible? Thanks.

Reply:
@vivianQizhu Done, thanks.